react/class_components/class_based_components.md

```diff
@@ -286,4 +286,4 @@ The following questions are an opportunity to reflect on key topics in this less
 
 This section contains helpful links to related content. It isn't required, so consider it supplemental.
 
-- In general, the React documentation on [thinking in react](https://legacy.reactjs.org/docs/thinking-in-react.html) is a really good place to look into, if you want more practice with class-based components, from scratch.
+- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
```

Comment:
```suggestion
- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.

```
Adds newline character to the bottom of the file to fix the lint warning.

Reply:
Hi @JoshDevHub, I have added a new line to the bottom of the file as suggested and sent a pull request. Appologies for the oversight.
Kind regards,
Amal